internal/operators/manager_test.go

```diff
@@ -390,21 +406,40 @@ var _ = Describe("Operators manager", func() {
 
 			Expect(err).To(BeNil())
 		})
-		It("error on operators supports both ARM and x86 while ARM architecture", func() {
-			monitoredOperators := []*models.MonitoredOperator{
-				{Name: "lvm"},
-				{Name: "lso"},
-				{Name: "odf"},
-				{Name: "cnv"},
-			}
-			cluster.CPUArchitecture = common.ARM64CPUArchitecture
-			err := manager.EnsureOperatorArchCapability(cluster, cluster.CPUArchitecture, monitoredOperators)
-
-			Expect(err).To(Not(BeNil()))
-			ExpectWithOffset(1, err.Error()).To(ContainSubstring("Local Storage Operator"))
-			ExpectWithOffset(1, err.Error()).To(ContainSubstring("OpenShift Data Foundation"))
-			ExpectWithOffset(1, err.Error()).To(ContainSubstring("OpenShift Virtualization"))
-			ExpectWithOffset(1, err.Error()).To(Not(ContainSubstring("Logical Volume Management")))
+		Context("error on operators supports both ARM and x86 while ARM architecture", func() {
+			table.DescribeTable("Validate  LVM with CNV on architecture", func(ocpVersion string, containsCNV bool) {
```

Comment:
Again, how lvm is related to the test? Should it be arm?
And in this case the test is not CNV only. Maybe - "validate ARM with different operators".

Reply:
same it's the manager which check each operator separately.
  
 I think you are missing some logic here. let talk about it off line